WebRoot/js/zimbraMail/share/model/ZmZimbraAccount.js

```diff
@@ -123,7 +123,7 @@ function() {
 	var identity = this.getIdentity();
 	var name = (!identity)
 		? this.settings.get(ZmSetting.DISPLAY_NAME)
-		: identity.name;
+		: AjxStringUtil.htmlDecode(identity.name);
```

Comment:
htmlDecode should not be done by us, it's responsibility of browser

Reply:
I have some places that we are using AjxStringUtil.htmlDecode()